appscale/tools/utils.py

```diff
@@ -272,6 +276,24 @@ def queues_from_xml(contents):
 
   return queues
 
+def dispatch_from_yaml(source_location):
+  dispatch_rules = None
+  with open(source_location) as config_file:
+    dispatch_rules = yaml.safe_load(config_file.read())
```

Comment:
Super minor, but just as a tip, `yaml.safe_load` accepts the file descriptor object as an argument so you don't need to read it all in first.